src/Components/Patient/ManagePatients.tsx

```diff
@@ -941,14 +956,14 @@ export const PatientManager = () => {
       </div>
       <div>
         <PatientFilter {...advancedFilter} key={window.location.search} />
-        <NavTabs
+        {/* <NavTabs
           onChange={(tab) => updateQuery({ is_active: tab ? "False" : "True" })}
           options={[
             { value: 0, label: "Live" },
             { value: 1, label: "Discharged" },
           ]}
           active={tabValue}
-        />
+        /> */}
```

Comment:
@GokulramGHV feel free to remove them if we are not using them or if they are not needed anymore